MonkeyWrench.Web.WebService/WebServices.asmx.cs

```diff
@@ -2633,234 +2633,187 @@ public GetBuildInfoResponse GetBuildInfoMultiple (WebServiceLogin login, string
 
 				lanes = db.GetAllLanes ();
 
-				switch (response.Host.QueueManagement) {
-				case DBQueueManagement.OneRevisionWorkAtATime:
```

Comment:
Done. I refactored the strategies into their own file since they were getting big.